### ISSUE TITLE
Bump Documenter version

### ISSUE THREAD

**Issue by mforets** [JuliaReach/LazySets.jl#2101]
```julia
 0.992525 seconds (1.06 M allocations: 54.281 MiB)
Test Summary:              | Pass  Total
LazySets.binary_operations |    4      4
 51.638593 seconds (67.83 M allocations: 3.545 GiB, 1.87% gc time)
WARNING: both IntervalRootFinding and TaylorIntegration export "derivative"; uses of it in module TaylorModels must be qualified
WARNING: both IntervalRootFinding and TaylorIntegration export "∇"; uses of it in module TaylorModels must be qualified
Test Summary:       | Pass  Total
LazySets.interfaces |   17     17
  1.531907 seconds (660.23 k allocations: 110.750 MiB, 1.26% gc time)
┌ Warning: received a polyhedron with no vertices during plotting
└ @ LazySets ~/.julia/dev/LazySets/src/Interfaces/AbstractPolyhedron_functions.jl:791
┌ Warning: received a polyhedron with no vertices during plotting
└ @ LazySets ~/.julia/dev/LazySets/src/Interfaces/AbstractPolyhedron_functions.jl:791
Test Summary:     | Pass  Total
LazySets.plotting |    2      2
 45.002608 seconds (80.80 M allocations: 3.823 GiB, 2.81% gc time)
[ Info: SetupBuildDirectory: setting up build directory.
[ Info: Doctest: running doctests.
┌ Error: doctest failure in ~/.julia/dev/LazySets/src/Sets/VPolytope.jl:28-31
│
│ ```jldoctest polytope_vrep
│ julia> P = VPolytope([0 0 0; 1 0 0; 0 1 0; 0 0 1])
│ VPolytope{Int64,Array{Int64,1}}(Array{Int64,1}[[0, 1, 0, 0], [0, 0, 1, 0], [0, 0, 0, 1]])
│ ```
│
│ Subexpression:
│
│ P = VPolytope([0 0 0; 1 0 0; 0 1 0; 0 0 1])
│
│ Evaluated output:
│
│ VPolytope{Int64,Array{Int64,1}}([[0, 1, 0, 0], [0, 0, 1, 0], [0, 0, 0, 1]])
│
│ Expected output:
│
│ VPolytope{Int64,Array{Int64,1}}(Array{Int64,1}[[0, 1, 0, 0], [0, 0, 1, 0], [0, 0, 0, 1]])
│
│   diff = VPolytope{Int64,Array{Int64,1}}(Array{Int64,1}[[0, VPolytope{Int64,Array{Int64,1}}([[0, 1, 0, 0], [0, 0, 1, 0], [0, 0, 0, 1]])
└ @ Documenter.DocTests ~/.julia/packages/Documenter/QQWIJ/src/DocTests.jl:369
┌ Error: doctest failure in ~/.julia/dev/LazySets/src/Sets/VPolytope.jl:36-45
│
│ ```jldoctest polytope_vrep
│ julia> M = [0 0 0; 1 0 0; 0 1 0; 0 0 1]'
│ 3×4 LinearAlgebra.Adjoint{Int64,Array{Int64,2}}:
│  0  1  0  0
│  0  0  1  0
│  0  0  0  1
│
│ julia> VPolytope(M)
│ VPolytope{Int64,Array{Int64,1}}(Array{Int64,1}[[0, 0, 0], [1, 0, 0], [0, 1, 0], [0, 0, 1]])
│ ```
│
│ Subexpression:
│
│ VPolytope(M)
│
│ Evaluated output:
│
│ VPolytope{Int64,Array{Int64,1}}([[0, 0, 0], [1, 0, 0], [0, 1, 0], [0, 0, 1]])
│
│ Expected output:
│
│ VPolytope{Int64,Array{Int64,1}}(Array{Int64,1}[[0, 0, 0], [1, 0, 0], [0, 1, 0], [0, 0, 1]])
│
│   diff = VPolytope{Int64,Array{Int64,1}}(Array{Int64,1}[[0, VPolytope{Int64,Array{Int64,1}}([[0, 0, 0], [1, 0, 0], [0, 1, 0], [0, 0, 1]])
└ @ Documenter.DocTests ~/.julia/packages/Documenter/QQWIJ/src/DocTests.jl:369
┌ Error: doctest failure in ~/.julia/dev/LazySets/src/LazyOperations/CartesianProductArray.jl:470-477
│
│ ```jldoctest
│ julia> using LazySets: block_to_dimension_indices
│
│ julia> cpa = CartesianProductArray([BallInf(zeros(n), 1.0) for n in [1, 3, 2, 3]]);
│
│ julia> block_to_dimension_indices(cpa, [2, 4, 8])
│ (Tuple{Int64,Int64}[(-1, -1), (2, 4), (-1, -1), (7, 9)], 2)
│ ```
│
│ Subexpression:
│
│ block_to_dimension_indices(cpa, [2, 4, 8])
│
│ Evaluated output:
│
│ ([(-1, -1), (2, 4), (-1, -1), (7, 9)], 2)
│
│ Expected output:
│
│ (Tuple{Int64,Int64}[(-1, -1), (2, 4), (-1, -1), (7, 9)], 2)
│
│   diff = (Tuple{Int64,Int64}[(-1, ([(-1, -1), (2, 4), (-1, -1), (7, 9)], 2)
└ @ Documenter.DocTests ~/.julia/packages/Documenter/QQWIJ/src/DocTests.jl:369
┌ Error: doctest failure in ~/.julia/dev/LazySets/src/Sets/VPolygon.jl:36-39
│
│ ```jldoctest polygon_vrep
│ julia> P = VPolygon([[0, 0], [1, 0], [0, 1]])
│ VPolygon{Int64,Array{Int64,1}}(Array{Int64,1}[[0, 0], [1, 0], [0, 1]])
│ ```
│
│ Subexpression:
│
│ P = VPolygon([[0, 0], [1, 0], [0, 1]])
│
│ Evaluated output:
│
│ VPolygon{Int64,Array{Int64,1}}([[0, 0], [1, 0], [0, 1]])
│
│ Expected output:
│
│ VPolygon{Int64,Array{Int64,1}}(Array{Int64,1}[[0, 0], [1, 0], [0, 1]])
│
│   diff = VPolygon{Int64,Array{Int64,1}}(Array{Int64,1}[[0, VPolygon{Int64,Array{Int64,1}}([[0, 0], [1, 0], [0, 1]])
└ @ Documenter.DocTests ~/.julia/packages/Documenter/QQWIJ/src/DocTests.jl:369
┌ Error: doctest failure in ~/.julia/dev/LazySets/src/Sets/VPolygon.jl:44-52
│
│ ```jldoctest polygon_vrep
│ julia> M = [0 1 0; 0 0 1.]
│ 2×3 Array{Float64,2}:
│  0.0  1.0  0.0
│  0.0  0.0  1.0
│
│ julia> VPolygon(M)
│ VPolygon{Float64,Array{Float64,1}}(Array{Float64,1}[[0.0, 0.0], [1.0, 0.0], [0.0, 1.0]])
│ ```
│
│ Subexpression:
│
│ VPolygon(M)
│
│ Evaluated output:
│
│ VPolygon{Float64,Array{Float64,1}}([[0.0, 0.0], [1.0, 0.0], [0.0, 1.0]])
│
│ Expected output:
│
│ VPolygon{Float64,Array{Float64,1}}(Array{Float64,1}[[0.0, 0.0], [1.0, 0.0], [0.0, 1.0]])
│
│   diff = VPolygon{Float64,Array{Float64,1}}(Array{Float64,1}[[0.0, VPolygon{Float64,Array{Float64,1}}([[0.0, 0.0], [1.0, 0.0], [0.0, 1.0]])
└ @ Documenter.DocTests ~/.julia/packages/Documenter/QQWIJ/src/DocTests.jl:369
┌ Error: doctest failure in ~/.julia/dev/LazySets/src/Sets/VPolytope.jl:93-105
│
│ ```jldoctest
│ julia> v = VPolytope()
│ VPolytope{Float64,Array{Float64,1}}(Array{Float64,1}[])
│
│ julia> dim(v)
│ -1
│
│ julia> v = VPolytope([ones(3)])
│ VPolytope{Float64,Array{Float64,1}}(Array{Float64,1}[[1.0, 1.0, 1.0]])
│
│ julia> dim(v) == 3
│ true
│ ```
│
│ Subexpression:
│
│ v = VPolytope([ones(3)])
│
│ Evaluated output:
│
│ VPolytope{Float64,Array{Float64,1}}([[1.0, 1.0, 1.0]])
│
│ Expected output:
│
│ VPolytope{Float64,Array{Float64,1}}(Array{Float64,1}[[1.0, 1.0, 1.0]])
│
│   diff = VPolytope{Float64,Array{Float64,1}}(Array{Float64,1}[[1.0, VPolytope{Float64,Array{Float64,1}}([[1.0, 1.0, 1.0]])
└ @ Documenter.DocTests ~/.julia/packages/Documenter/QQWIJ/src/DocTests.jl:369
┌ Error: doctest failure in ~/.julia/dev/LazySets/src/Approximations/template_directions.jl:270-278
│
│ ```jldoctest; filter = r"2246[0-9]*e-16"
│ julia> using LazySets.Approximations: PolarDirections
│
│ julia> pd = PolarDirections(2)
│ PolarDirections{Float64}(2, Array{Float64,1}[[1.0, 0.0], [-1.0, 1.22465e-16]])
│
│ julia> pd.Nφ
│ 2
│ ```
│
│ Subexpression:
│
│ pd = PolarDirections(2)
│
│ Evaluated output:
│
│ PolarDirections{Float64}(2, [[1.0, 0.0], [-1.0, 1.2246467991473532e-16]])
│
│ Expected output:
│
│ PolarDirections{Float64}(2, Array{Float64,1}[[1.0, 0.0], [-1.0, 1.22465e-16]])
│
│   diff = PolarDirections{Float64}(2, Array{Float64,1}[[1.0, [[1.0, 0.0], [-1.0, 1.22465e-16]])1.2246467991473532e-16]])
└ @ Documenter.DocTests ~/.julia/packages/Documenter/QQWIJ/src/DocTests.jl:369
┌ Error: doctest failure in ~/.julia/dev/LazySets/src/Approximations/template_directions.jl:349-357
│
│ ```jldoctest spherical_directions; filter = r"1232[0-9]*e-17.*2246[0-9]*e-16.*1232[0-9]*e-17"
│ julia> using LazySets.Approximations: SphericalDirections
│
│ julia> sd = SphericalDirections(3)
│ SphericalDirections{Float64}(3, 3, Array{Float64,1}[[0.0, 0.0, 1.0], [0.0, 0.0, -1.0], [1.0, 0.0, 6.12323e-17], [-1.0, 1.22465e-16, 6.12323e-17]])
│
│ julia> sd.Nθ, sd.Nφ
│ (3, 3)
│ ```
│
│ Subexpression:
│
│ sd = SphericalDirections(3)
│
│ Evaluated output:
│
│ SphericalDirections{Float64}(3, 3, [[0.0, 0.0, 1.0], [0.0, 0.0, -1.0], [1.0, 0.0, 6.123233995736766e-17], [-1.0, 1.2246467991473532e-16, 6.123233995736766e-
17]])
│
│ Expected output:
│
│ SphericalDirections{Float64}(3, 3, Array{Float64,1}[[0.0, 0.0, 1.0], [0.0, 0.0, -1.0], [1.0, 0.0, 6.12323e-17], [-1.0, 1.22465e-16, 6.12323e-17]])
│
│   diff = SphericalDirections{Float64}(3, 3, Array{Float64,1}[[0.0, [[0.0, 0.0, 1.0], [0.0, 0.0, -1.0], [1.0, 0.0, 6.12323e-17], 6.123233995736766e-17], [-1.
0, 1.22465e-16, 6.12323e-17]])1.2246467991473532e-16, 6.123233995736766e-17]])
└ @ Documenter.DocTests ~/.julia/packages/Documenter/QQWIJ/src/DocTests.jl:369
┌ Error: Doctesting failed
│   exception =
│    `makedocs` encountered a doctest error. Terminating build
│    Stacktrace:
│     [1] error(::String) at ./error.jl:33
│     [2] runner(::Type{Documenter.Builder.Doctest}, ::Documenter.Documents.Document) at /home/mforets/.julia/packages/Documenter/QQWIJ/src/Builder.jl:204
│     [3] dispatch(::Type{Documenter.Builder.DocumentPipeline}, ::Documenter.Documents.Document) at /home/mforets/.julia/packages/Documenter/QQWIJ/src/Utiliti
es/Selectors.jl:167
│     [4] #2 at /home/mforets/.julia/packages/Documenter/QQWIJ/src/Documenter.jl:237 [inlined]
│     [5] cd(::Documenter.var"#2#3"{Documenter.Documents.Document}, ::String) at ./file.jl:104
│     [6] makedocs(; debug::Bool, format::Documenter.Writers.HTMLWriter.HTML, kwargs::Base.Iterators.Pairs{Symbol,Any,NTuple{5,Symbol},NamedTuple{(:root, :sou
rce, :sitename, :doctest, :modules),Tuple{String,String,String,Symbol,Array{Module,1}}}}) at /home/mforets/.julia/packages/Documenter/QQWIJ/src/Documenter.jl:
236
│     [7] (::Documenter.var"#all_doctests#30"{Bool,Array{Module,1}})() at /home/mforets/.julia/packages/Documenter/QQWIJ/src/Documenter.jl:725
│     [8] macro expansion at /home/mforets/.julia/packages/Documenter/QQWIJ/src/Documenter.jl:741 [inlined]
│     [9] macro expansion at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Test/src/Test.jl:1113 [inlined]
│     [10] doctest(::String, ::Array{Module,1}; fix::Bool, testset::String) at /home/mforets/.julia/packages/Documenter/QQWIJ/src/Documenter.jl:741
│     [11] doctest(::Module; manual::Bool, testset::Nothing, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/mforets/.j
ulia/packages/Documenter/QQWIJ/src/Documenter.jl:684
│     [12] doctest(::Module) at /home/mforets/.julia/packages/Documenter/QQWIJ/src/Documenter.jl:671
│     [13] top-level scope at /home/mforets/.julia/dev/LazySets/test/runtests.jl:172
│     [14] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Test/src/Test.jl:1113
│     [15] top-level scope at /home/mforets/.julia/dev/LazySets/test/runtests.jl:172
│     [16] top-level scope at util.jl:175
│     [17] top-level scope at /home/mforets/.julia/dev/LazySets/test/runtests.jl:172
│     [18] include(::Module, ::String) at ./Base.jl:377
│     [19] exec_options(::Base.JLOptions) at ./client.jl:288
│     [20] _start() at ./client.jl:484
└ @ Documenter ~/.julia/packages/Documenter/QQWIJ/src/Documenter.jl:734
Doctests: LazySets: Test Failed at /home/mforets/.julia/packages/Documenter/QQWIJ/src/Documenter.jl:741
  Expression: all_doctests()
Stacktrace:
 [1] macro expansion at /home/mforets/.julia/packages/Documenter/QQWIJ/src/Documenter.jl:741 [inlined]
 [2] macro expansion at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Test/src/Test.jl:1113 [inlined]
 [3] doctest(::String, ::Array{Module,1}; fix::Bool, testset::String) at /home/mforets/.julia/packages/Documenter/QQWIJ/src/Documenter.jl:741
Test Summary:        | Fail  Total
LazySets.doctests    |    1      1
  Doctests: LazySets |    1      1
ERROR: LoadError: Some tests did not pass: 0 passed, 1 failed, 0 errored, 0 broken.
in expression starting at /home/mforets/.julia/dev/LazySets/test/runtests.jl:169
[mforets@localhost LazySets]$ julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.4.0 (2020-03-21)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(@v1.4) pkg> add Documenter#v0.23.4
   Updating registry at `~/.julia/registries/General`
   Updating git-repo `https://github.com/JuliaRegistries/General.git`
  Resolving package versions...
   Updating `~/.julia/environments/v1.4/Project.toml`
  [e30172f5] ↓ Documenter v0.24.7 ⇒ v0.23.4 #v0.23.4 (https://github.com/JuliaDocs/Documenter.jl.git)
   Updating `~/.julia/environments/v1.4/Manifest.toml`
  [e30172f5] ↓ Documenter v0.24.7 ⇒ v0.23.4 #v0.23.4 (https://github.com/JuliaDocs/Documenter.jl.git)

(@v1.4) pkg>

```

**Comment by mforets** [comment]
some of these failures may only show up when using LazySets with julia v1.4.

**Comment by schillic** [comment]
This was resolved in #2151.
